### ISSUE TITLE
Installation error: Following authentication credentials are required: Missing required header field: Authorization

### ISSUE THREAD

**Issue by mlutfy** [syxys1/me.sxs.square#1]
I ran `cv ext:enable square`, and stumbled on this error:

```
Error: API Call Failed: Array
(
    [entity] => Extension
    [action] => install
    [params] => Array
        (
            [keys] => Array
                (
                    [0] => me.sxs.square
                )

            [debug] => 1
            [version] => 3
        )

    [result] => Array
        (
            [trace] => #0 wp-content/uploads/civicrm/ext/square/vendor/apimatic/core/src/Authentication/Auth.php(107): Core\Exceptions\AuthValidationException::init()
#1 wp-content/uploads/civicrm/ext/square/vendor/apimatic/core/src/Client.php(121): Core\Authentication\Auth->validate()
#2 wp-content/uploads/civicrm/ext/square/vendor/apimatic/core/src/Request/RequestBuilder.php(145): Core\Client->validateAuth()
#3 wp-content/uploads/civicrm/ext/square/vendor/apimatic/core/src/ApiCall.php(45): Core\Request\RequestBuilder->build()
#4 wp-content/uploads/civicrm/ext/square/vendor/square/square/src/Apis/BaseApi.php(34): Core\ApiCall->execute()
#5 wp-content/uploads/civicrm/ext/square/vendor/square/square/src/Apis/WebhookSubscriptionsApi.php(86): Square\Apis\BaseApi->execute()
#6 wp-content/uploads/civicrm/ext/square/CRM/square/squareUtils.php(125): Square\Apis\WebhookSubscriptionsApi->listWebhookSubscriptions()
#7 wp-content/uploads/civicrm/ext/square/CRM/square/squareUtils.php(105): myListWebhookSubscriptions()
#8 wp-content/uploads/civicrm/ext/square/square.php(65): myUpdateWebhookSubscription()
#9 civicrm/civicrm/CRM/Extension/Manager/Module.php(71): square_civicrm_postinstall()
[...]
#27 {main}
            [is_error] => 1
            [error_message] => Following authentication credentials are required:
-> Missing required header field: Authorization
        )
)
```

**Comment by mlutfy** [comment]
ah, I see it's maybe because of this, mentioned in the README:

> Before installation, you need to configure your SQUARE_ACCESS_TOKEN as an environment variable.
Under, nginx, this is done in /etc/nginx/fastcgi.config.

**Comment by mlutfy** [comment]
Fixed in #2
